### ISSUE TITLE
Rest After doesn't increase for not used "tickets"

### ISSUE THREAD

**Issue by NyuRailgun** [TiiaAurora/LS-public-issue-tracker#40]
The Rest After doesn't show the increas of Rest Bonus of the day if you do not use one or more "entries".

Rest After in the Daily section calculates correctly if you use the collected Rest Bonus from day before.
Rest Before from the next day also shows the correct number if you do not use any entries of the day.
![image](https://user-images.githubusercontent.com/52236837/167472469-d7a41b48-13c2-4d7a-8fa0-bed0cac6b83f.png)

**Comment by TiiaAurora** [comment]
The "Rest After" should always match the amount you see in the Ingame UI. The calculations for after the reset will be shown in Rest before for the next day regardless if you did everything or nothing or just partial.

**Comment by NyuRailgun** [comment]
> The "Rest After" should always match the amount you see in the Ingame UI. The calculations for after the reset will be shown in Rest before for the next day regardless if you did everything or nothing or just partial. We split this on purpose because we only did the calculation before and people were confused why the rest amount didnt match the ingame amount shown.

But isn't that the function of "rest before" ? rest before = rest bonus at start of the day, rest after = rest at the end of the day ?
weird ppl minds :(

**Comment by coldino** [comment]
After = after you've done what you ticked, so it always matches the UI.

Before = at the very start of the day, after getting the rest from the end of the previous day. You don't get this rest until the reset time, so it's never useful to include it in "After".

This way people are free to override what they see at the start of the day (Before) or after they've done stuff (After) without having to juggle with additional bonus that won't be applied until reset time.

Previously we only showed the number you talk about (including the rest you would get at reset time), and without exception it confused absolutely everyone we talked to. In particular losing 30 points from ticking a box blew people's minds, but it is totally correct. Splitting the fields makes this much more understandable.
